src/DynamoPackages/PackageDirectoryBuilder.cs

```diff
@@ -222,7 +220,7 @@ internal void CopyFilesIntoRetainedPackageDirectory(IEnumerable<IEnumerable<stri
             {
                 // We expect that files are bundled in root folders
                 // For single files, just get its folder
-                var commonPath = files.Count() > 1 ? GetLongestCommonPrefix(files.ToArray()) : Path.GetDirectoryName(files.First());
+                var commonPath = files.Count() > 1 ? GetLongestCommonPrefix(files.ToArray()) : Path.GetDirectoryName(files.FirstOrDefault());
```

Comment:
files.Any()

Reply:
files.Any() won't work here since we need to know if there is only 1 item vs more than 1